### ISSUE TITLE
At 200% zoom, broken underline on `Read about our approach` text in the Footer

### ISSUE THREAD

**Issue by jtart** [bbc/simorgh#863]
**Describe the bug**
When the page is zoomed in to 200%, in Chrome at some breakpoints - When you hover/focus on ‘Read about our approach…’ the underline covers some of the text.

This issue can also be seen on iPhone 5 when using VoiceOver (some users use a keyboard with a mobile device).

**To Reproduce**
View on Chrome in desktop, and on iPhone 5, view a page at 200% zoom, and view the underline covering some of the text in the `Read about our approach...` in the Footer.

**Expected behaviour**
Underline should not cover text, likely should discuss with UX about how it should look.

**Desktop (please complete the following information):**

- Chrome

**Smartphone (please complete the following information):**

- iPhone 5

**Comment by BogdanDogaru** [comment]
Blocked on #828 and #616

**Comment by sareh** [comment]
Still valid issue:

_Screen Shot of footer with 200% zoom_
<img width="492" alt="Screen Shot of footer with 200% zoom" src="https://user-images.githubusercontent.com/3028997/55793046-e93c3f00-5ab9-11e9-98a0-6542e2ea28ee.png">

Note: even without zooming, if you have a small viewport, the underline overlaps the text below:

Screen Shot of footer with 100% zoom, in a small viewport (which allows link to flow over multiple lines)
<img width="247" alt="Screen Shot of footer with 100% zoom" src="https://user-images.githubusercontent.com/3028997/55793215-518b2080-5aba-11e9-9b16-f59fce3fb0a0.png">

Should be raised with @nataliesmart from UX.

**Comment by sareh** [comment]
As a result of discussing with @greenc05 & @nataliesmart, it was discussed to change the hover/focus to use `text-decoration: underline` instead of the border bottom. 

Here's the impact of before/after when changing this in the browser:

Current implementation with `border-bottom`:
<img width="91" alt="Screen Shot 2019-04-09 at 13 16 20" src="https://user-images.githubusercontent.com/3028997/55799722-fa8d4780-5ac9-11e9-8e73-681b6965a5e6.png">
<img width="211" alt="Screen Shot 2019-04-09 at 13 19 46" src="https://user-images.githubusercontent.com/3028997/55799814-2e686d00-5aca-11e9-85c3-e824489f1255.png">


With `text-decoration: underline`
<img width="105" alt="Screen Shot 2019-04-09 at 13 16 41" src="https://user-images.githubusercontent.com/3028997/55799721-fa8d4780-5ac9-11e9-836b-09dbef8f9dea.png">
<img width="211" alt="Screen Shot 2019-04-09 at 13 15 56" src="https://user-images.githubusercontent.com/3028997/55799723-fa8d4780-5ac9-11e9-9470-0eae0bda53d7.png">

**Comment by sareh** [comment]
Added label `simorgh-core-stream` since all front-end components that are shared between Articles and Front Pages are to be addressed by Simorgh Core.

**Comment by jamesbhobbs** [comment]
Covered here https://github.com/bbc/psammead/issues/397
